samples/bluetooth/central_bas/README.rst

```diff
@@ -20,7 +20,8 @@ Requirements
 
 * One of the following development boards:
 
-  * nRF9160 DK board (PCA10090)
+  * |nRF9160DK|
+  * |nRF5340DK|
   * nRF52840 Development Kit board (PCA10056)
```

Comment:
Would it make sense to add these other DKs to shortcuts as well? It would make it easier to track them in all the docs.

Reply:
Yes, but this is not in the scope for this PR. We can do it in NCSDK-3600.